<map id="pylo.vendors.urllib3.exceptions.ReadTimeoutError" name="pylo.vendors.urllib3.exceptions.ReadTimeoutError">
<area shape="rect" id="node1" title=" " alt="" coords="872,39,1071,80"/>
<area shape="rect" id="node2" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_timeout_error.html" title=" " alt="" coords="625,5,824,47"/>
<area shape="rect" id="node3" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_h_t_t_p_error.html" title=" " alt="" coords="132,39,331,80"/>
<area shape="rect" id="node6" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_pool_error.html" title=" " alt="" coords="379,68,577,109"/>
<area shape="rect" id="node4" title=" " alt="" coords="5,46,84,73"/>
<area shape="rect" id="node5" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_request_error.html" title=" " alt="" coords="625,71,824,112"/>
</map>
